app/views/layouts/to_change_everything.html.erb

```diff
@@ -0,0 +1,263 @@
+<% cache @locale, expires_in: 1.year do %>
```

Comment:
this gives a huge performance boost when I test locally, but I see no difference in staging...